src/StdCheats.sol

```diff
@@ -463,6 +464,15 @@ abstract contract StdCheatsSafe {
             vm.resumeGasMetering();
         }
     }
+
+    // a cheat for fuzzing addresses that are payable only
+    // see https://github.com/foundry-rs/foundry/issues/3631
+    function assumePayable(address addr) internal virtual {
+        try PanicAssertions.assertPayable(addr) {}
```

Comment:
Clever approach! LGTM, let's just remove this unnecessary PanicAssertions library and inline the call here instead

Reply:
totally agree the PanicAssertions library seems unnecessary but unfortunately the try/catch statement "can only catch errors from external function calls and contract creation." - [link](https://solidity-by-example.org/try-catch/)

if you propose a better place to put the external function, i can move it

Reply:
would a low level call work instead? 

```solidity
(bool success, ) = payable(addr).call{value: 0}("");
vm.assume(success);
```

if not, maybe an external function in the same contract and call it with `this.isPayable(addr)` with a comment explaining that method's purpose so user's know they don't need to call it

Reply:
looks like low level call works, thanks

[9dbf1a4](https://github.com/foundry-rs/forge-std/pull/270/commits/9dbf1a47505bc7a6dd8a203f52e8d7592c759d0c)